src/backend/marsha/core/serializers/video.py

```diff
@@ -325,6 +325,7 @@ class Meta:  # noqa
             "is_ready_to_show",
         )
 
+    starting_at = serializers.DateTimeField(required=False, allow_null=True)
```

Comment:
AS we are using a `ModelSerializer` do we have to redeclare this ? Any reason to do it ?

Reply:
no, you right I've left it by mistake, it was tests with my timezone problemem